src/app/sb-admin/service/user.service.ts

```diff
@@ -40,4 +40,22 @@ export class UserService {
     })
     return this.http.post(environment.userCreateUrl, payload, { headers: header });
   }
+
+  blockUser(payload: any): Observable<any> {
```

Comment:
use only one service to block and unblock

Reply:
resolved